ops/model.py

```diff
@@ -38,6 +38,8 @@
 
 logger = logging.getLogger(__name__)
 
+MAX_STR_LEN = 131071  # Max length of strings to pass to subshell.
```

Comment:
I wonder if this could be renamed, perhaps `MAX_SUBSHELL_STR_LEN` or similar - this feels a bit of a generic name to me